includes/class-wc-ajax.php

```diff
@@ -1536,6 +1540,15 @@ public static function json_search_products( $term = '', $include_variations = f
 		$include_ids = ! empty( $_GET['include'] ) ? array_map( 'absint', (array) wp_unslash( $_GET['include'] ) ) : array();
 		$exclude_ids = ! empty( $_GET['exclude'] ) ? array_map( 'absint', (array) wp_unslash( $_GET['exclude'] ) ) : array();
 
+		if ( ! empty( $_GET['exclude_type'] ) ) {
+			$exclude_types = array_map( 'strtolower', (array) wp_unslash( $_GET['exclude_type'] ) ); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```

Comment:
We could replace this code to:
```php
$exclude_types = array_map( 'trim', array_map( 'strtolower', explode( ',', wp_unslash( $_GET['exclude_type'] ) ) ) )
```

Reply:
I've added support for comma delimited lists.